<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>diffractionImaging</class>
 <widget class="QWidget" name="diffractionImaging">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Diffraction imaging</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QPushButton" name="addFiles">
           <property name="text">
            <string>Add files</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="sortButton">
           <property name="text">
            <string>Sort</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QListView" name="listFiles"/>
       </item>
       <item>
        <widget class="QCheckBox" name="multiframe">
         <property name="text">
          <string>Multiframe</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Number of frames</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="numberOfFrames">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="1">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="2" colspan="2">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Number of points</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Fast motor</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="fastMotorName">
           <property name="text">
            <string>Translation</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QLineEdit" name="fastMotorPts"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Low motor</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="slow_motor_name">
           <property name="text">
            <string>Rotation</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QLineEdit" name="slowMotorPts"/>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Offset</string>
           </property>
          </widget>
         </item>
         <item row="3" column="3">
          <widget class="QLineEdit" name="offset"/>
         </item>
         <item row="4" column="0" colspan="3">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Number of points in experiment</string>
           </property>
          </widget>
         </item>
         <item row="4" column="3">
          <widget class="QLineEdit" name="totalPts">
           <property name="toolTip">
            <string>fast_pts * slow_pts - offset</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Azimuthal integration</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="configureDiffraction">
           <property name="text">
            <string>Configure</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Output file</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="outputFile"/>
         </item>
         <item>
          <widget class="QPushButton" name="outputFileSelector">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QPushButton" name="run">
           <property name="text">
            <string>Run</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="progressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="abort">
           <property name="text">
            <string>Abort</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
